flatpak/theia.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
```

Comment:
let's add `set -xeuo pipefail` here per the Unofficial Bash Strict Mode recommendations (http://redsymbol.net/articles/unofficial-bash-strict-mode/)